src/messages.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+/******************************************************************************\
+ * Copyright (c) 2022
+ * Author(s):
+ *  Peter Goderie (pgScorpio)
+ *
+ *  Use this static class to show basic Error, Warning and Info messages
+ *
+ *  For own created special message boxes you should always use
+ *      CMessages::MainForm() as parent parameter
+ *  and CMessages::MainFormName() as base of the title parameter
+\******************************************************************************/
+
+#include <QString>
+#ifndef HEADLESS
+#    include <QMessageBox>
+#    define tMainform QDialog
+#else
+#    include <QDebug>
```

Comment:
What does QDebug include? Does it add bloat on a release build?

Reply:
QDebug includes what's neccesary for the cli streams.